src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/GetTaskByIdControllerTest.java

```diff
@@ -93,4 +93,83 @@ public void should_return_a_200_with_task_and_correct_properties() {
         common.cleanUpTask(taskId);
     }
 
+
+    @Test
+    public void should_return_403_when_user_grant_type_standard_and_excluded() {
```

Comment:
could be renamed as should_return_403_when_user_has_grant_type_standard_and_excluded

Reply:
I think one more FT challenged and excluded could be added